packages/react-mui-store-file-viewer/src/components/CreateNewFolderButton/fields.tsx

```diff
@@ -78,6 +82,7 @@ export const getCreateFolderFields = ({
     label: "Advanced Settings",
     collaspable: true,
     defaultOpen: false,
+    isHidden: () => hideFolderOptions,
```

Comment:
Why are the advanced settings hidden>

Reply:
This was from Bruce, he wanted a simple create folder modal.

Reply:
Okay